design-reference/docs/azure-simple.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Azure Simple
+draft: true
+date: 2019-05-18T13:09:18.252Z
```

Comment:
Can this be automated?

Reply:
Yes, it should be scaffolded out by the CLI. `title` will come from user prompt, `draft` will always be set to `true`, and `date` will come from current date of CLI usage.